monai/apps/nuclick/transforms.py

```diff
@@ -8,19 +8,42 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 import math
 import random
+from enum import Enum
+from typing import Union
 
 import numpy as np
 
 from monai.config import KeysCollection
-from monai.transforms import MapTransform, RandomizableTransform, SpatialPad
+from monai.transforms import MapTransform, Randomizable, SpatialPad
 from monai.utils import optional_import
 
 measure, _ = optional_import("skimage.measure")
 morphology, _ = optional_import("skimage.morphology")
 
 
+class NuclickKeys(Enum):
```

Comment:
(str, Enum) this way you directly use NuclickKeys.CENTROID instead of NuclickKeys.CENTROID.value

Reply:
interesting workaround for the type checking...I guess it becomes a problem if there's a mixture of `NuclickKeys.IMAGE` and `"image"`  in a transform chain...

Reply:
this is a good one, let me create a separate feature request for this, thanks @SachidanandAlle